test/Fullcount.t.sol

```diff
@@ -1626,3 +1635,185 @@ contract FullcountTest_unstake is FullcountTestBase {
         vm.stopPrank();
     }
 }
+
+contract FullcountTest_inviteOnly is FullcountTestBase {
+    uint256 SessionID;
+    address PitcherNFTAddress;
+    uint256 PitcherTokenID;
+    address BatterNFTAddress;
+    uint256 BatterTokenID;
+
+    function test_as_batter() public {
+        charactersMinted++;
+        uint256 tokenID = charactersMinted;
+
+        otherCharactersMinted++;
+        uint256 otherTokenID = otherCharactersMinted;
+
+        characterNFTs.mint(player1, tokenID);
+        otherCharacterNFTs.mint(player2, otherTokenID);
+
+        uint256 initialNumSessions = game.NumSessions();
+
+        uint256 initialBlockTimestamp = block.timestamp;
+        uint256 startJoinOffsetSeconds = 5;
+        uint256 expectedNextPhaseTimestamp = initialBlockTimestamp + startJoinOffsetSeconds;
+
+        vm.startPrank(player1);
+
+        uint256 sessionID = game.startSession(address(characterNFTs), tokenID, PlayerType.Pitcher, true);
```

Comment:
I don't see tests that joining a session fails with a signature that's invalid in some way.

Reply:
(If the session requires signatures)